### ISSUE TITLE
mIOU is low

### ISSUE THREAD

**Issue by BandaryWang** [jfzhang95/pytorch-deeplab-xception#24]
Thanks for sharing the code.
When I used this project to train Pascal VOC for nearly 200 epochs, I got only 0.32 mIOU.

The result on tensorboardX is:
![wrong_result](https://user-images.githubusercontent.com/25317905/48116681-cfe05e00-e2a1-11e8-928f-11550ceab2ff.jpg)

My configuration is:
learning rate=0.0001
backbone=xception
use_sbd=False

And I have set size_average=True in nn.CrossEntropyLoss function.

**Comment by jfzhang95** [comment]
I noticed that you set `use_sbd=False`, that means you only use VOC 2012 dataset to train the model. Therefore, maybe training images are insufficient. Pls look into #23 .

You can download SBD dataset and set `use_sbd=True` to train the model.

**Comment by BandaryWang** [comment]
Thanks for your reply.
I will use SBD dataset and train again.

**Comment by jfzhang95** [comment]
You're welcome.
